### PR TITLE
Add capabilities to TransactionOptions

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/src/wallet/operations/transaction/account.rs [hunk 1]
```diff
@@ -102,12 +103,17 @@ where
         // TODO https://github.com/iotaledger/iota-sdk/issues/1740
         let issuance = self.client().get_issuance().await?;
 
+        // TODO this needs to be removed when Mana is properly handled
```
Comment (Alex6323) [inline sdk/src/wallet/operations/transaction/account.rs]: issue so we don't forget?

**Review by DaughterOfMars — COMMENTED** [review]
are the capabilities actually used anywhere?

**Comment by thibault-martinez** [comment]
> are the capabilities actually used anywhere?

Apart from the temporary use, not right now no. But we still need support for it.

File: sdk/src/wallet/operations/transaction/account.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/src/wallet/operations/transaction/account.rs]: Done

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**